src/components/Header/DefaultHeader.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+
+import { ClosedLockIcon, UserIcon, LogOutIcon } from '@entur/icons'
+import { useToast } from '@entur/alert'
+
+import { TavlaLogo, Github } from '../../assets/icons'
+import LoginModal from '../LoginModal'
+
+import { useUser } from '../../auth'
+import firebase from 'firebase'
+
+export function DefaultHeader({ theme }: Props): JSX.Element {
+    const [displayLogin, setDisplayLogin] = useState<boolean>(false)
+    const user = useUser()
+    const userLoggedin = !user || !user.isAnonymous
+    const { addToast } = useToast()
+
+    const login = (): void => {
+        event.preventDefault()
+        setDisplayLogin(true)
+    }
+
+    const logout = (): void => {
+        event.preventDefault()
+        addToast({
+            title: 'Logget ut',
+            content: 'Du er nå logget ut av din konto',
```

Comment:
skal det være punktum i strengen kanskje?

Reply:
Dette er kopiert fra noe som sto på BottomMenu, men kan for så vidt endre på alle forekomster om det ser bedre ut